explorer/explorer_cluster.go

```diff
@@ -47,6 +49,9 @@ func (n *ClusterExplorer) Execute(selection string) error {
 			PreviousExplorer:   n,
 		}
 		return Explore(namespaceExplorer)
+	case exitLabel:
+		os.Exit(0)
```

Comment:
Can we pull the exit out into a function so we can call it from other areas of the code? Im working on a signal handler that would want to call said function

Reply:
niice. Ok will do